test/test_maskedtensor.py

```diff
@@ -96,6 +105,195 @@ def _fix_fn_name(fn_name):
     return fn_name
 
 
+class TestBasics(TestCase):
+    def test_add(self):
```

Comment:
this is a negative test of (I think) incompatible masks, name needs to reflect what is being tested. also idk if this means add() isn't being positive tested but if not that should be added

Reply:
added the positive test!